### PR TITLE
Documentation update for readthedocs

### PR DESCRIPTION
**Pull request by kcho**
This is the PR for the `readthedocs` documentation linked below. 

https://amp-scz-lochness.readthedocs.io/en/kcho-readthedocs/

### CODE REVIEW

**Comment by tashrifbillah** [comment]
Hi @kcho , are #65, #66, #67 independent of each other? If not what should be the sequence of review?

**Comment by kcho** [comment]
> 

Yeap they are independent to each other. #66 and #67 first, please. Thanks.

**Comment by tashrifbillah** [comment]
Did you add textured background to [amp_scz_logo.png](/amp_scz_logo.png) as opposed to the plain one I provided?

**Comment by kcho** [comment]
is it not the one you provided?

File: requirements.txt [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+cryptease>=0.2.0
+pytz>=2021.1
+requests>=2.26.0
+six>=1.16.0
+pandas>=1.3.2
+pytest>=6.2.4
+numpy>=1.20.3
+mano>=0.5.1
+LAMP>=0.0.1
+PyYAML>=6.0
```
Comment (tashrifbillah) [inline requirements.txt:14]: I see that this file is a copy of the packages in `setup.py`. Why is this one required then? Regardless, updating packages in two places will be burdensome in future. This is how it is done for convenience:

https://github.com/pnlbwh/conversion/blob/48f94ce21005d1af779f98df0c5ea8ec0e204835/setup.py#L9-L18

```python
requirements= open('requirements.txt').read().split()

setup(name='conversion',
    version=__version__,
    ...
    install_requires=requirements,

```

I can help with it.

**Comment by tashrifbillah** [comment]
> is it not the one you provided? it should be the one you provided I see no 'AMP' in the logo image.

Sorry about the false alarm. All good. I realize that GitHub diff shows it with a textured background:

image

**Comment by kcho** [comment]
oh I have removed white background from the original image to make it appear nice in the readthedocs website.

File: scripts/lochness_create_template.py [hunk 2]
```diff
@@ -225,22 +225,38 @@ def create_keyring_template(keyring_loc: Path, args: object) -> None:
...
+                mindlamp_ak = '*****'
+                mindlamp_sk = '*****'
+
             template_dict[f'mindlamp.{study}'] = {
-                "URL": "**api.lamp.digital**",
-                "ACCESS_KEY": args.email,
-                "SECRET_KEY": "**PASSWORD**"}
+                "URL": mindlamp_url,
+                "ACCESS_KEY": mindlamp_ak,
+                "SECRET_KEY": mindlamp_sk}
```
Comment (tashrifbillah) [inline scripts/lochness_create_template.py:259]: So in the above, are we accepting user inputs for passwords before creating the template config?

Comment (kcho) [reply scripts/lochness_create_template.py:259]: if they use `--enter-password` option, `getpass` is used to safely transfer user-entered information to the output template.

File: docs/source/conf.py [hunk 3]
```diff
@@ -74,7 +74,9 @@
 # The theme to use for HTML and HTML Help pages.  See the documentation for
 # a list of builtin themes.
 #
-html_theme = 'alabaster'
+# html_theme = 'alabaster'
+html_theme = 'sphinx_rtd_theme'
+html_logo = 'images/AMP SCZ Logo small.png'
```
Comment (tashrifbillah) [inline docs/source/conf.py:79]: Are single quotes around a file name with spaces okay?

**Comment by tashrifbillah** [comment]
These files are all new to me:

* .readthedocs.yml
* docs/requirements.txt
* docs/source/conf.py

So there is not much to review.

File: requirements.txt (hunk 1 above)
Comment (tashrifbillah) [reply requirements.txt:14]: @kcho 

> Why is this one required then?

File: docs/source/conf.py (hunk 3 above)
Comment (kcho) [reply docs/source/conf.py:79]: yeap all works.

**Comment by kcho** [comment]
> These files are all new to me:

These are the configuration files required for the `sphinx` and `readthedocs` to work smoothly with our documentation configurations.

File: requirements.txt (hunk 1 above)
Comment (tashrifbillah) [reply requirements.txt:14]: Based on my discussion with Kevin, `requirements.txt` is redundant and so it should be removed.

**Pull request merged**